CRM/Core/Payment.php

```diff
@@ -518,9 +519,23 @@ public function getText($context, $params) {
             $gotText .= ' ' . ts('You will receive an email receipt for each recurring contribution.');
           }
         }
-        break;
+        return $gotText;
+
+      case 'contributionPageContinueText':
+        if ($params['amount'] <= 0) {
+          return ts('To complete this transaction, click the <strong>Continue</strong> button below.');
+        }
+        if ($this->_paymentProcessor['billing_mode'] == 4) {
+          return ts('Click the <strong>Continue</strong> button to go to %1, where you will select your payment method and complete the contribution.', [$this->_paymentProcessor['payment_processor_type']]);
+        }
+        if ($params['is_payment_to_existing']) {
```

Comment:
Payment to existing what?  I can't find this parameter anywhere in core?

Reply:
it's defined just in the call to this function as a parameter - it seemed like a fairly clear variable name